build_tools/pkgci/generate_benchmark_graphs.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python3
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+
+import argparse
+import json
+import tempfile
+import zipfile
+
+from pathlib import Path
+from typing import List, Dict
+
+from gh_utils import (
+    list_gh_artifacts,
+    fetch_gh_artifact,
+    get_commit_from_run_id,
+)
+
+benchmark_workflows = ["torch_models_cpu_task", "torch_models_amdgpu_mi325"]
+
+max_history_length = 100
```

Comment:
100 commits is a bit less i think. We commit around ~30-40 commits a week. It might be worth having atleast a month of data. Maybe 200?